teams.md

```diff
@@ -0,0 +1,117 @@
+
+* Workflow automation bots [#8116](https://github.com/gravitational/teleport/pull/8116)
+* Designing and building next generation CI and CD platforms.
+
+### Teleport Devops
+
+Teleport DevOps is responsible for building, maintaining, and testing open-source modules for various configuration management platforms. This will be the open-source code that enables both our OSS and Enterprise customers to deploy and maintain Teleport on their chosen configuration management system. We’re looking to cover Kubernetes, Terraform on various clouds, Ansible, Salt, Puppet, etc.
+
+## Cloud Platform team
+
+The Teleport Cloud offering is new; we're on a journey of defining and building a culture around production, offering a secure and reliable hosted version of Teleport Enterprise as a service.
+
+Our mandate is straightforward; we need to be prepared to secure, monitor, maintain, scale, investigate, and automate our production environment for Teleport Cloud. Our work includes a wide variety of projects drawing on various skill sets, from ensuring we're up to date on patches to redesigning parts of the Teleport Core project to be more appealing to the Cloud.
+
+#### Here are some of the projects we're working on right now:
+
+* Rewriting the Teleport core project networking so that agents run by customers require fewer connections and overhead when phoning home.
+* We are expanding Teleport Cloud to multiple regions, reducing latency between users and their infrastructure.
+* We are deploying auto scalers, so no one needs to be paged to scale our infrastructure when there is a burst in new accounts.
+* We're improving our security posture, currently focusing on ensuring we have an audit trail for generated and published assets and ensuring we're using encryption at rest in the right places.
+* Progressing beyond minimum viable monitoring and alerting and fully adopting Grafana Cloud as our centralized metrics, alerting and logging platform.
+
+#### The toolset we currently use is:
+
+* Golang
+* Terraform and Packer
+* Kubernetes
+* Prometheus / Alertmanager / Loki internally and on Grafana Cloud
+* Amazon Web Services
+* Drone.io
+* and More
+
+## Cloud Application team
+
+Teleport Cloud is the SaaS version of Teleport that we launched in early 2021. As the applications team for Teleport Cloud, our goal is to manage the deployments of Teleport for our cloud customers and provide a stable, easy-to-use product.
+
+The team intersects with almost all other departments in the company and as such there is a large variety of projects that we're involved in, for example:
+
+* Custom Kubernetes operators
+* Subscription & billing logic
+* Internal dashboards & admin panels
+* UI & UX improvements for Teleport
+* Analytics & Monitoring
+* Asset distribution
+
+#### We use the following stack:
+
+* Go
+* Typescript
+* React
+* Kubernetes
+* AWS
+* Terraform
+* PostgreSQL
+* gRPC
+* Grafana & Prometheus
```

Comment:
@knisbet Can you review these parts and tell me if it's an accurate description of what your team does?

Reply:
Yea, although with the team merge/split, should probably be updated into the 4 I have now.